composer.json

```
@@ -20,7 +20,7 @@
     ],
     "require": {
         "php": "^8.2",
-        "hammerstone/sidecar": "^0.4.0",
+        "hammerstone/sidecar": "^0.5.0",
```

I still want to support Laravel 10, so lets update the constraint like this.

```suggestion
        "hammerstone/sidecar": "^0.4 || ^0.5",
```